install/kubernetes/istio-rbac-alpha.yaml

```diff
@@ -1,14 +1,109 @@
 # Permissions and roles for istio
+# To debug: start the cluster with -vmodule=rbac,3 to enable verbose logging on RBAC DENY
+# Also helps to enable logging on apiserver 'wrap' to see the URLs.
+# Each RBAC deny needs to be mapped into a rule for the role.
+# If using minikube, start with '--extra-config=apiserver.Authorization.Mode=RBAC'
+#
+# NOTE: If deploying istio to a namespace other than 'default' then change the
+# ClusterRoleBinding namspace target appropriately.
+kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1alpha1
-kind: RoleBinding
 metadata:
-  name: istio-binding
+  name: istio-manager
+rules:
+- apiGroups: ["istio.io"]
+  resources: ["istioconfigs", "istioconfigs.istio.io"]
+  verbs: ["*"]
+- apiGroups: ["extensions"]
+  resources: ["thirdpartyresources", "thirdpartyresources.extensions", "ingresses"]
+  verbs: ["*"]
+- apiGroups: [""]
+  resources: ["configmaps", "endpoints", "pods", "services", "namespaces"]
```

Comment:
`2017-05-15T18:23:14.640098266Z I0515 18:23:14.639846       1 config.go:41] Parsed route-rule productpage-default into istio.proxy.v1.config.RouteRule destination:"productpage.istio.svc.cluster.local" precedence:1 route:<tags:<key:"version" value:"v1" > > 
2017-05-15T18:23:14.640140163Z I0515 18:23:14.639899       1 handler.go:78] Adding config to Istio registry: key istio/route-rule-productpage-default, config &{Type:route-rule Name:productpage-default Spec:map[route:[map[tags:map[version:v1]]] destination:productpage.istio.svc.cluster.local precedence:1] ParsedSpec:destination:"productpage.istio.svc.cluster.local" precedence:1 route:<tags:<key:"version" value:"v1" > > }
2017-05-15T18:23:14.642366289Z W0515 18:23:14.642132       1 handler.go:205] namespace "istio" not present`